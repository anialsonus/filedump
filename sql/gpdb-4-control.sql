-- control file
\! pg_filedump -c test/pg_control.gp4
