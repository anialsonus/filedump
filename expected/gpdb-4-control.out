-- control file
\! pg_filedump -c test/pg_control.gp4

*******************************************************************
* PostgreSQL File/Block Formatted Dump Utility
*
* File: test/pg_control.gp4
* Options used: -c
*******************************************************************

<pg_control Contents> *********************************************

                          CRC: Correct
           pg_control Version: 8220430
              Catalog Version: 201310150
            System Identifier: 6789779736718376126
                        State: IN PRODUCTION
                Last Mod Time: Thu Feb  6 09:52:09 2020
       Last Checkpoint Record: Log File (0) Offset (0x05952300)
   Previous Checkpoint Record: Log File (0) Offset (0x05951670)
  Last Checkpoint Record Redo: Log File (0) Offset (0x05952300)
          |-       TimeLineID: 1
          |-         Next XID: 0/1273
          |-         Next OID: 40961
          |-       Next Multi: 1
          |-    Next MultiOff: 0
          |-             Time: Thu Feb  6 09:52:09 2020
       Minimum Recovery Point: Log File (0) Offset (0x00000000)
          Backup Start Record: Log File (0) Offset (0x00000000)
End-of-Backup Record Required: no
       Maximum Data Alignment: 8
        Floating-Point Sample: 1234567
          Database Block Size: 32768
           Blocks Per Segment: 32768
              XLOG Block Size: 32768
            XLOG Segment Size: 67108864
    Maximum Identifier Length: 64
           Maximum Index Keys: 32
   Date and Time Type Storage: 64 bit Integers
         Locale Buffer Length: 128
                   lc_collate: en_US.utf8
                     lc_ctype: en_US.utf8

